database/migrations/2022_02_22_072517_create_books.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBooks extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('books', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->bigInteger('isbn');
```

Comment:
ISBN numarası için Semih bir review yapmıştı, ISBN numarası sıfır ile başlayabileceğini söylemişti. Bu durumda sıfır ile başlayan bir değer girildiğinde burası bir tam sayı değeri alacağından ve tam sayı sıfır ile başlayamayacağıdan yanlış değer alacak. Burası bigInteger yerine string olmalı. Semih'in yaptığı review'ı tekrar incele